editors/code/package.json

```diff
@@ -520,6 +520,11 @@
                         "type": "boolean",
                         "default": false
                     },
+                    "rust-analyzer.debug.buildBeforeRestart": {
+                        "markdownDescription": "Whether to rebuild the project modules before debugging the same test again",
```

Comment:
is there ever a reason not wanting to have this behavior?

Reply:
This was added to make the behavior more flexible. I can imagine there could be use cases where you want to debug code without it rebuilding! Why should it be fixed?

Reply:
> I can imagine there could be use cases where you want to debug code without it rebuilding!

I don't think so. If you changed the source, the debugger is going to be worse than useless (i.e. misleading) unless you only look at the assembly (and never at the source code, watches etc.).

I think this should at least default to `true`.